src/cmds/qdel.c

```diff
@@ -58,6 +58,99 @@
 #include <pbs_config.h>   /* the master config generated by configure */
 #include <pbs_version.h>
 
+#define NO_OF_JOBIDS 1000 /* Using this macro to allocate array of pointers */
```

Comment:
In your earlier approach, you were sending jobs by batches to the server if it is a huge array.  That will give room for server to do other useful work. Are we achieving the same now,  if not why?

Reply:
I have brought back the initial approach I followed that is, sending jobs in batches. Now, in each request, the initial number of jobs based on the email suppression number and the following iteration will be sending 10k jobs.